<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/OpenCOR.qrc">
    <normaloff>:/appIcon</normaloff>:/appIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionSystem"/>
     <addaction name="separator"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionFrench"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="separator"/>
    <addaction name="actionResetAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionHomePage"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
     <addaction name="actionHelpToolbar"/>
    </widget>
    <addaction name="menuToolbars"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="helpToolbar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit OpenCOR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/en.png</normaloff>:/en.png</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
   <property name="statusTip">
    <string>Select English as the language used by OpenCOR</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/fr.png</normaloff>:/fr.png</iconset>
   </property>
   <property name="text">
    <string>&amp;French</string>
   </property>
   <property name="statusTip">
    <string>Select French as the language used by OpenCOR</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/help.png</normaloff>:/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="statusTip">
    <string>Look up the OpenCOR help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/home.png</normaloff>:/home.png</iconset>
   </property>
   <property name="text">
    <string>Home &amp;Page</string>
   </property>
   <property name="statusTip">
    <string>Look up the OpenCOR home page</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../res/OpenCOR.qrc">
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="statusTip">
    <string>Some general information about OpenCOR</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionResetAll">
   <property name="text">
    <string>&amp;Reset All</string>
   </property>
   <property name="statusTip">
    <string>Reset all the user settings</string>
   </property>
  </action>
  <action name="actionHelpToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="statusTip">
    <string>Show/hide the Help toolbar</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;System</string>
   </property>
   <property name="statusTip">
    <string>Select the system's language as the language used by OpenCOR</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/OpenCOR.qrc"/>
 </resources>
 <connections/>
</ui>
